content/docs/start/model-management/model-registry.md

```diff
@@ -0,0 +1,185 @@
+---
+title: 'Get Started: Model registry'
+description:
+  'Set up a git-based model registry with DVC to track and manage models, their
+  versions and lifecycle stages.'
+---
+
+# Get Started: Model Registry
+
+Just as we use experiment tracking to manage model development, it is a good
+idea to keep a model registry to manage the lifecycle of the models we get from
+our experiments. Using DVC and Studio we will set up a model registry where we
+can discover, share, deploy and audit all our models and which will serve as the
+single source of truth for our model management.
+
+To make use of our model registry, we will need some trained models first. To
+speed things up, we will start from a git
+[repository](https://github.com/iterative/example-get-started-model-management)
+with a model training pipeline already set up and ready to use.
+
+To see how you can track experiments and set up training pipelines with DVC and
+DVCLive, have a look at our getting started guide for
+[experiment management](/doc/start/experiments).
+
+## DVC Model registry overview
```

Comment:
I think we could drop this section so we can get to the code quicker.

Reply:
We could move some of the info down below where you show how to see the model in Studio.

Reply:
It might also be worth explaining somewhere that Studio is convenient and the only way to have a registry across multiple repos, but that the underlying functionality can be done with dvc and gto.